docs/reference/commandline/cli.md

```diff
@@ -48,6 +48,8 @@ by the `docker` command line:
 * `DOCKER_TLS_VERIFY` When set Docker uses TLS and verifies the remote.
 * `DOCKER_CONTENT_TRUST` When set Docker uses notary to sign and verify images.
   Equates to `--disable-content-trust=false` for build, create, pull, push, run.
+* `DOCKER_CONTENT_TRUST_SERVER` The URL of the Notary server to use. This defaults
```

Comment:
It's documented on the trust sandbox page. If someone does need to use it, this just makes it easier to find. Also, currently it's the only way to use your own notary with the open source registry unless you set up a load balancer in front of it. I think it's useful information. I don't think we should remove it without deprecating it first either way, so including it here wouldn't make that any harder.